test/test_docstrings.py

```diff
@@ -37,6 +37,15 @@ def test_rst_extract_options_fuzz(s):
     assert not s or content or options
 
 
+def test_convert_exception():
```

Comment:
This is not a useful test because it tests unwanted behavior (we don't want any known cases of raising). Let's use pytest's monkeypatch to replace `rst` with a method that raises an exception, and then test with that.

Reply:
Ahhh I see, that makes more sense.

Reply:
Does the new commit look good now?